src/locale/id/amo.js

```diff
@@ -20,32 +20,32 @@ module.exports = { domain:"messages",
       "View desktop site":[ "Lihat situs desktop" ],
       "Browse in your language":[ "Jelajahi dalam bahasa Anda" ],
       "Firefox Add-ons":[ "Pengaya Firefox" ],
-      "How are you enjoying your experience with %(addonName)s?":[ "" ],
-      "screenshot %(imageNumber)s of %(totalImages)s":[ "" ],
-      "Average rating: %(rating)s out of 5":[ "" ],
-      "No ratings":[ "" ],
-      "%(users)s user":[ "" ],
-      "Log out":[ "" ],
+      "How are you enjoying your experience with %(addonName)s?":[ "Bagaimana pengalaman Anda menggunakan %(addonName)s?" ],
```

Comment:
Yeah, because the locale files were causing errors and needed to be rebuilt to fix them; I figured I'd include them here. Otherwise this won't show as fixed until we rebuilt the locales which won't be until next week.